### ISSUE TITLE
Feedback - Broken Image Issue in the 'Remember Me' Checkbox Section

### ISSUE THREAD

**Issue by huzaifaalmesbah** [WordPress/Learn#2345]
## Type of feedback
//content

## Description
The example image in the 'Remember Me' Checkbox section at the bottom of the page is broken. The image fails to load, and only alt text is visible.

## Step-by-step reproduction instructions (optional)
1. Go to https://learn.wordpress.org/lesson/how-to-log-in-with-wp-admin
2. Scroll down to the bottom of the page
3. Locate the 'Remember Me' Checkbox section

## Screenshots or screen recording (optional)
![Huzaifa-20240324130608](https://github.com/WordPress/Learn/assets/63150399/c9bd7276-897b-4b28-be1c-e2ca97d0a8cc)

**Comment by huzaifaalmesbah** [comment]
![Huzaifa-20240324131554](https://github.com/WordPress/Learn/assets/63150399/79508525-3b37-4587-b684-d7adc3caf882)

**Comment by ghost** [comment]
Updated image. Checked front end. All looks good.
